scripts/generate-dev-overrides.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+BIN_DIR=$PWD/bin
+
+cat << EOF > $BIN_DIR/developer_overrides.tfrc
```

Comment:
I'd prefer to shift this name into a variable as it's used three times.

Reply:
Updated the script to use a variable when referencing the overrides filename.